flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/committer/FilesCommittable.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.flink.sink.committer;
+
+import java.io.Serializable;
+import org.apache.iceberg.io.WriteResult;
+import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
+
+public class FilesCommittable implements Serializable {
+  private final WriteResult committable;
+  private String jobID = "";
+  private long checkpointId;
+  private int subtaskId;
+
+  public FilesCommittable(WriteResult committable) {
+    this.committable = committable;
+  }
+
+  public FilesCommittable(WriteResult committable, String jobID, long checkpointId, int subtaskId) {
+    this.committable = committable;
+    this.jobID = jobID;
+    this.checkpointId = checkpointId;
+    this.subtaskId = subtaskId;
+  }
+
+  public WriteResult committable() {
+    return committable;
+  }
+
+  public Long checkpointId() {
+    return checkpointId;
+  }
+
+  public String jobID() {
```

Comment:
Do we always use capital letters for `jobID`, and different capitalisation for other id-s, like `checkpointId` - not really important, but strange to see it handled differently with other id-s

Reply:
The usage of Flink is followed here. 😄

Reply:
Let me modify it. It is also ok to use `jobId`.